docs/components/content/examples/vue/collapsible/ExampleVueCollapsibleBasic.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+const isOpen = ref(false)
+</script>
+
+<template>
+  <NCollapsible
+    v-model:open="isOpen"
+    class="w-350px space-y-2"
+  >
+    <div class="flex items-center justify-between px-4 space-x-4">
+      <h4 class="text-sm font-semibold">
+        @peduarte starred 3 repositories
+      </h4>
+      <NCollapsibleTrigger as-child>
```

Comment:
maybe we will ue the `<template #trigger></template>` ? and with content too

Reply:
@mnenie yes we can, but this time this is different as it override all slots since we a default slot.

ref:
![Screenshot 2024-10-11 at 12 28 46 AM](https://github.com/user-attachments/assets/a426cfb6-3843-48c2-8c4c-a19efae0a8a0)

Reply:
The anatomy and example are based on Radix UI and shadcn-vue.